.github/workflows/build_and_test_release_latest.yml

```diff
@@ -1,54 +1,31 @@
-name: Build & Test utils_common release-latest
```

Comment:
To check you can add `pull_request` to this patch to verify your changes will work before merging. After verifying I would then remove.

Reply:
Done, passed. reverting now